src/server/game/Entities/Creature/CreatureData.h

```diff
@@ -114,14 +114,15 @@ enum class CreatureRandomMovementType : uint8
 struct TC_GAME_API CreatureMovementData
 {
     CreatureMovementData() : Ground(CreatureGroundMovementType::Run), Flight(CreatureFlightMovementType::None), Swim(true), Rooted(false), Chase(CreatureChaseMovementType::Run),
-        Random(CreatureRandomMovementType::Walk) { }
+        Random(CreatureRandomMovementType::Walk), InteractionPauseTimer(180000) { }
```

Comment:
this default value might have to be changed

Reply:
Use the worldserver config default value here instead. Making default handling more flexible
`sWorld->getIntConfig(CONFIG_CREATURE_STOP_FOR_PLAYER)`